internal/example/example_validators.go

```diff
@@ -130,11 +132,181 @@ func (s Test) Validate() error {
 	// max
 
 	if len(s.MaxMultiple) > 7 {
-		vErrors = append(vErrors, gencheck.NewFieldError("Test", "MaxMultiple", "max", errors.New("length was more than 7")))
+		vErrors = append(vErrors, gencheck.NewFieldError("Test", "MaxMultiple", "max", errors.New("length failed check for max=7")))
 	}
 
 	// END MaxMultiple Validations
 
+	// BEGIN LtString Validations
```

Comment:
It's definitely generated code,  I'll see how hard an option like that would be.

The whitespace is difficult to work with in the templates, you either get it removed (including the line break) or you get extra spaces.  It's only the goimports that cleans up what gets printed out, and they leave in one line break.